web-wallet/src/lib/dusk/components/__tests__/Card.spec.js

```diff
@@ -5,16 +5,8 @@ import { Card } from "..";
 describe("Card", () => {
```

Comment:
I would also add a test for the default Card slot here until a helper is created to test the named slots.

Reply:
I'm not sure what the way to do this would be.

What I could do is to have a component that is the Card with some content, as a test component, and then check the snapshot of that in the test. Similar to what I wanted to do for the table on the Explorer. 👀